packages/flutter/lib/src/animation/animations.dart

```diff
@@ -76,6 +95,7 @@ class ProxyAnimation extends Animation<double>
   AnimationDirection _direction;
   double _value;
 
+  /// The animation whose value this animation will proxy.
```

Comment:
Mention that it's mutable and that the listeners will be appropriately forwarded.